api/src/main.ts

```diff
@@ -14,13 +14,15 @@ import { AppModule } from './modules/app.module';
 import { CustomExceptionFilter } from './utilities/custom-exception-filter';
 import { json } from 'express';
 import { logger } from './middleware/logger.middleware';
+import {  WinstonModule } from 'nest-winston';
```

Comment:
Based on this line it seems like you may not have prettier working locally, it integrates directly into most editors, lmk if you want support setting it up so that our formatting can stay consistent!

Reply:
interesting. I have it but it didn't throw anything i'll take a look